includes/event/event-capabilities.php

```diff
@@ -240,6 +252,45 @@ private function has_edit_attendees( WP_User $user, Event $event ): bool {
 		return false;
 	}
 
+	/**
+	 * Evaluate whether a user can edit event title for a specific event.
+	 *
+	 * @param WP_User $user  User for which we're evaluating the capability.
+	 * @param Event   $event Event for which we're evaluating the capability.
+	 * @return bool
+	 */
+	private function has_edit_form_field( WP_User $user, Event $event, $cap ): bool {
+		$event_start         = $event->start();
```

Comment:
We could drop this variable and call `$event->start()` directly whenever it's needed.